frontend/src/pages/beginner.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { type ReactNode, useState } from "react";
+import Image from "next/image";
+import NavbarMain from "@/common/components/NavBarMain";
+import Footer from "@/common/components/Footer";
+
+const styles = {
+  container: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+    height: "100vh",
+  },
+  btn: {
+    appearance: "button",
+    backgroundColor: "#000",
+    border: "1px solid #000",
+    borderRadius: "4px",
+    boxShadow: "4px 4px 0 0 rgba(255, 255, 255, 1), 4px 4px 0 1px #000",
+    boxSizing: "border-box",
+    color: "#fff",
+    cursor: "pointer",
+    display: "inline-block",
+    fontFamily: "ITCAvantGardeStd-Bk, Arial, sans-serif",
+    fontSize: "14px",
+    fontWeight: "400",
+    lineHeight: "20px",
+    padding: "12px 40px",
+    textAlign: "center",
+    textTransform: "none",
+    touchAction: "manipulation",
+    userSelect: "none",
+    WebkitUserSelect: "none",
+    whiteSpace: "nowrap",
+  },
+  msg: {
+    marginTop: "20px",
+    fontFamily: "Pacifico, cursive",
+    fontSize: "24px",
+  },
+};
```

Comment:
good that you included the css here for scope purposes. Normally, we would want to centralize our css into a `.css` file, but I didn't expect that for this task. Cool style/font by the way